doc/pod_chaos.md

```diff
@@ -0,0 +1,70 @@
+# Pod Chaos Document
+
+This document helps you to build pod chaos experiments. 
+
+Pod chaos allows you to simulate pod faults, specifically `pod failure` and `pod kill`.
```

Comment:
Add a reminder that we do not currently support simulation injection of naked pods, only support some specific pods, such as deployment 、statefulset、daemonset

Reply:
I have added the note.